src/components/SignUpForm/index.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import { Button, Grid, LinearProgress } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { TextField } from 'formik-material-ui';
+
+import { Formik, Field, Form } from 'formik';
+import * as Yup from 'yup';
+
+import styles from './styles';
+import MajorDropdown from '../Dropdowns/MajorDropdown';
```

Comment:
add index.js in Dropdowns to export both so we only need 1 import statement
also change Dropdowns to lowercase to adhere with naming conventions throughout codebase

Reply:
Wait but I thought the convention for src/components' naming of folders is capitalization of first letter of all words in a name? That's what I see for other folders. Am I missing something here?

Reply:
yes that's true - and perhaps I shouldn't have said naming conventions throughout codebase.

My rationale was more along the lines of React codebases having only first letter capitalized for components - since dropdowns is a **folder** of components it shouldnt be capitalized

Reply:
Oh I see now. Thank you so much for clarifying! Will change it accordingly